tests/kw_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+. ./tests/utils --source-only
+. ./kw.sh --source-only
+
+function suite
+{
+  suite_addTest "testVariables"
+  suite_addTest "testExported"
+}
+
+function testVariables
+{
+  VARS=( EASY_KERNEL_WORKFLOW src_script_path external_script_path )
```

Comment:
We assumed `src_script_path` and `external_script_path` were not exported to the user. That's why we merely sourced `kw.sh` and expected `src_script_path` and `external_script_path` to be defined, and only tested `EASY_KERNEL_WORKFLOW` in `testExported`. 

https://github.com/rodrigosiqueira/kworkflow/blob/7f8f56f82a23eff2b5bc5ab3cf2aa6a16103bf84/tests/kw_test.sh#L4

This is not the case it seems. A quick test on my bash shows `kw` actually exports all three variables. This could potentially be a problem akin to #16.